go.mod

```diff
@@ -41,3 +43,5 @@ require (
 )
 
 replace github.com/ethereum/go-ethereum v1.9.14 => github.com/vulcanize/go-ethereum v1.10.14-statediff-0.0.29
+
+replace github.com/vulcanize/go-eth-state-node-iterator => ../state-node-iterator
```

Comment:
This should be replaced before merging

Reply:
@roysc if you push up your branch of the state node iterator with the mocks pkg, I can resolve the dep issue, perform some local benchmarks and get this merged.